csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -1487,6 +1488,250 @@ void scheduleInnerOuterPersistentKernel(
   inlineMost();
 }
 
+void scheduleTmaWarpSpecializedOuter(
```

Comment:
`scheduleTmaWarpSpecializedOuter()` is a duplication of `scheduleReductionCombinedOuter()` with some clean ups e.g. removing code handling `rparams->multiple_reds_per_blk` since it will not be used for TMA warp specialized kernel.